db/migrate/20151104115400_create_chargeback_tiers.rb

```diff
@@ -0,0 +1,13 @@
+class CreateChargebackTiers < ActiveRecord::Migration
+  def change
+    create_table :chargeback_tiers do |t|
+      t.bigint :chargeback_rate_detail_id
+      t.float :start
+      t.float :end
+      t.float :fix_rate
+      t.float :var_rate
+
+      t.timestamp :null => false
+    end
+  end
+end
```

Comment:
This migration is going to need to handle migrating existing rate values the chargeback_rate_details table to the chargeback_rate_tiers table before the rate column is removed. Otherwise, existing rates will be lost.